docs/commands/Add-ShouldOperator.mdx

```diff
@@ -35,7 +35,7 @@ This function allows you to create custom Should assertions.
 function BeAwesome($ActualValue, [switch] $Negate)
 {
 
-    [bool] $succeeded = $ActualValue -eq 'Awesome'
+[bool] $succeeded = $ActualValue -eq 'Awesome'
```

Comment:
```suggestion
    [bool] $succeeded = $ActualValue -eq 'Awesome'
```
Weird formatting issue mulitple code examples. The first line after a blank line loses indentation. If you remove line 37, then line 41 will get the same issue. @bravo-kernel Thoughts?